tests/test_thread.cpp

```diff
@@ -191,6 +191,86 @@ class threadpool_t : public dnnl::threadpool_interop::threadpool_iface {
 } // namespace testing
 } // namespace dnnl
 
+#elif defined(DNNL_TEST_THREADPOOL_USE_EIGEN_ASYNC)
+
+// absl sources define its own version of `CHECK` macro. oneDNN's version is not
+// needed further the file, thus, disable it for compilation reason.
+#undef CHECK
+
+#define EIGEN_USE_THREADS
+
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+
+#include "Eigen/ThreadPool"
+
+#include "xla/backends/cpu/runtime/work_queue.h"
+#include "xla/tsl/concurrency/async_value_ref.h"
+#include "xla/tsl/concurrency/chain.h"
+
+#define EIGEN_USE_THREADS
```

Comment:
is defining `EIGEN_USE_THREADS` twice required..?

Reply:
Thanks for spotting, updated the section.